src/groupeddataframe/splitapplycombine.jl

```diff
@@ -407,31 +245,34 @@ const KWARG_PROCESSING_RULES =
     in addition to those generated. In this case if the returned
     value contains columns with the same names as the grouping columns, they are
     required to be equal.
+
+    If `ungroup=true` (the default) a `DataFrame` is returned.
+    If `ungroup=false` a `GroupedDataFrame` grouped using `keycols(gdf)` is returned.
     """
 
 """
-    combine(gd::GroupedDataFrame, args...; keepkeys::Bool=true)
-    combine(fun::Union{Function, Type}, gd::GroupedDataFrame; keepkeys::Bool=true)
-    combine(pair::Pair, gd::GroupedDataFrame; keepkeys::Bool=true)
-    combine(gd::GroupedDataFrame, fun::Union{Function, Type}; keepkeys::Bool=true)
-    combine(gd::GroupedDataFrame, pair::Pair; keepkeys::Bool=true)
+    combine(gd::GroupedDataFrame, args...; keepkeys::Bool=true, ungroup::Bool=true)
+    combine(fun::Union{Function, Type}, gd::GroupedDataFrame;
+            keepkeys::Bool=true, ungroup::Bool=true)
+    combine(pair::Pair, gd::GroupedDataFrame; keepkeys::Bool=true, ungroup::Bool=true)
+    combine(fun::Union{Function, Type}, df::AbstractDataFrame, ungroup::Bool=true)
+    combine(pair::Pair, df::AbstractDataFrame, ungroup::Bool=true)
 
-Transform a [`GroupedDataFrame`](@ref) into a `DataFrame`.
+Apply operations to each group in a [`GroupedDataFrame`](@ref) and return
+the combined result as a `DataFrame`.
```

Comment:
I guess `ungroup` should be mentioned here directly, or the type of the result shouldn't be mentioned here?

Reply:
Ah - OK. I have updated both `combine` and `select`.